### ISSUE TITLE
Overvote modal similar to mocks

### ISSUE THREAD

**Issue by benadida** [votingworks/bmd#118]

**Comment by beausmith** [comment]
Charlie feedback: Language on change-vote modal is too complicated.

**Comment by beausmith** [comment]
On single-seat contest, optionally provide alternate modal buttons to:
- unselect "CURRENT CANDIDATE" and select "NEW CANDIDATE"
- unselect all

**Comment by beausmith** [comment]
@mcchilders says:

> I also have a language question about the “you must unselect before you can select again” modal - is there plain-language guidance on action language for touchscreens anywhere? People unfamiliar with touchscreen devices isn’t a new prob, so hopefully yes? Knowing that “select” in the instructions means tap/touch is maybe intuitive, maybe not, but I’d wager that knowing “unselect” means tap/touch what you already selected again (instead of tapping outside the selections, like you would in many mouse-driven interfaces) is pretty rare in certain populations?
> 
> And answering my own question - turns out Dana has already done this, from back when they were testing DREs (starting on p74 you get to the interesting findings): https://ws680.nist.gov/publication/get_pdf.cfm?pub_id=901388

**Comment by beausmith** [comment]
From Charlie's user testing: 

Tester returns to a contest to view here vote. The contest only has one candidate and it is selected. She asks, "Is there only one?" She then returns to the review screen, then back to the contest screen, and still isn't sure, but convinces herself that there is only one option.

This is a reason to do two things:
1. Update the info line above the candidates to add the number
2. Disable instead of hide the write-in option when total seats are met on a contest.

**Comment by benadida** [comment]
I agree we should make this dialog better, I leave it to @beausmith to make the specific call.
